3.Promises-draw-circle/draw-circle-task-3/script.js

```diff
@@ -0,0 +1,20 @@
+function showCircle(cx, cy, radius) {
+  let div = document.createElement("div");
+  div.style.width = 0;
+  div.style.height = 0;
+  div.style.left = cx + "px";
+  div.style.top = cy + "px";
+  div.className = "circle";
+  document.body.append(div);
+
+  return new Promise((resolve) => {
+    setTimeout(() => {
```

Comment:
why do we need `setTimeout` here?

Reply:
Because the transition doesn't work without it and when we click the circle comes up without any transition animations. Is there any better way to solve this?

Reply:
no, that's fine